servers/rendering/shader_language.cpp

```diff
@@ -2742,13 +2742,13 @@ bool ShaderLanguage::_validate_function_call(BlockNode *p_block, const FunctionI
 		//stage based function
 		const StageFunctionInfo &sf = p_function_info.stage_functions[name];
 		if (argcount != sf.arguments.size()) {
-			_set_error(vformat("Invalid number of arguments when calling stage function '%s', which expects %d arguments.", String(name), sf.arguments.size()));
+			_set_error(vformat(RTR("An invalid number of arguments when calling stage function '%s', which expects %d arguments."), String(name), sf.arguments.size()));
```

Comment:
The original was better actually:
```suggestion
			_set_error(vformat(RTR("Invalid number of arguments when calling stage function '%s', which expects %d arguments."), String(name), sf.arguments.size()));
```
"An" could be added but then it calls for a more complete sentence, like "An invalid number of arguments **was passed** when calling stage function '%s'...". Without it, it's just a nominal group, which is a common construct for error messages to avoid having to repeat the same verbs over and over (like "Got an invalid number of ....").